inference-engine/tests/functional/plugin/cpu/single_layer_tests/gru_cell.cpp

```diff
@@ -0,0 +1,129 @@
+
+class GRUCellCPUTest : public testing::WithParamInterface<GRUCellCpuSpecificParams>,
+                            virtual public LayerTestsUtils::LayerTestsCommon,
+                            public CPUTestsBase {
+public:
+    static std::string getTestCaseName(const testing::TestParamInfo<GRUCellCpuSpecificParams> &obj) {
+        CPUSpecificParams cpuParams;
+        LayerTestsDefinitions::GRUCellParams basicParamsSet;
+        std::map<std::string, std::string> enforceBF16;
+
+        std::tie(basicParamsSet, cpuParams, enforceBF16) = obj.param;
+
+        std::ostringstream result;
+        result << LayerTestsDefinitions::GRUCellTest::getTestCaseName(
+            testing::TestParamInfo<LayerTestsDefinitions::GRUCellParams>(basicParamsSet, 0));
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+
+        if (!enforceBF16.empty()) {
+            result << "_PluginConf";
+            for (auto &item : enforceBF16) {
+                if (item.second == PluginConfigParams::YES)
+                    result << "_" << item.first << "=" << item.second;
+            }
+        }
+        return result.str();
+    }
+
+protected:
+    void SetUp() {
+        CPUSpecificParams cpuParams;
+        LayerTestsDefinitions::GRUCellParams basicParamsSet;
+        std::map<std::string, std::string> enforceBF16;
+
+        bool should_decompose;
+        size_t batch;
+        size_t hidden_size;
+        size_t input_size;
+        std::vector<std::string> activations;
+        std::vector<float> activations_alpha;
+        std::vector<float> activations_beta;
+        float clip;
+        bool linear_before_reset;
+        InferenceEngine::Precision netPrecision;
+
+        std::tie(basicParamsSet, cpuParams, enforceBF16) = this->GetParam();
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        std::tie(should_decompose, batch, hidden_size, input_size, activations, clip, linear_before_reset, netPrecision, targetDevice) = basicParamsSet;
+
+        std::vector<std::vector<size_t>> inputShapes = {
+            {{batch, input_size},
+             {batch, hidden_size},
+             {3 * hidden_size, input_size},
+             {3 * hidden_size, hidden_size},
+             {(linear_before_reset ? 4 : 3) * hidden_size}},
+        };
+
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        auto params = ngraph::builder::makeParams(ngPrc, {inputShapes[0], inputShapes[1]});
+        std::vector<ngraph::Shape> WRB = {inputShapes[2], inputShapes[3], inputShapes[4]};
+        auto gru_cell = ngraph::builder::makeGRU(
+            ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes(params)), WRB, hidden_size, activations, {}, {}, clip, linear_before_reset);
+        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(gru_cell->output(0))};
+        function = std::make_shared<ngraph::Function>(results, params, "gru_cell");
```

Comment:
Suggest to use CPUTestsBase::makeNgraphFunction to add cpu specific stuff here and in all the other tests.